docs/resources/Poll.md

```diff
@@ -0,0 +1,122 @@
+# Poll Resource
+
+A poll is... well... a poll! It holds information about a poll!
+
+![If you get it you get it](example-poll.png)
+
+### Poll Object
+
+The poll object has a lot of levels and nested structures. It was also designed
+to support future extensibility, so some fields may appear to be more complex than
+necessary.
+
+The request and response models are closely aligned. So for the sake of brevity
+we will have common structure declarations instead of duplicating request and response structures.
+
+Fields marked by an \* are only sent as part of responses from Discord's API/Gateway.
+
+###### Poll Object Structure
+
+| Field             | Type                                                                                                | Description                                                     |
+|-------------------|-----------------------------------------------------------------------------------------------------|-----------------------------------------------------------------|
+| question          | [Poll Media Object](#DOCS_RESOURCES_POLL/poll-media-object-poll-media-object-structure)             | The question of the poll. Only `text` is supported.             |
+| answers           | List of [Poll Answer Objects](#DOCS_RESOURCES_POLL/poll-answer-object-poll-answer-object-structure) | Each of the answers available in the poll.                      |
+| expiry            | IS08601 timestamp                                                                                   | The time when the poll ends.                                    |
```

Comment:
I think they changed the `expiry` field to a `duration` field, which is just an integer that indicates how many hours the poll will stay up.

Reply:
I will correct this, thanks lol

Reply:
Both the `expiry` field and the old `duration` field suffer from the same problem of using time based math instead of calendar math. When datetime calculates extend >1 day you want to use calendar based math so it meets human expectations when e.g. DST transitions happen. Right now since this is a flat hour calculation you end up with the poll expiring +/- 1 hour from what you originally intended during these transitions.

Note that events suffer from the same issue as well but I'd rather this gets fixed here before shipping.

Reply:
We talked about this internally, for polls, and decided that this is a wontfix. 

We understand the problem, but we believe that the impact is low enough that it's not worth either leaking the author's timezone, or building UI to pick some timezone for this.

We acknowledge that in our UI it says "3 days", but semantically it means 72 hours.